Tests/XMLCoderTests/Auxiliary/XMLStackParserTests.swift

```diff
@@ -56,4 +56,106 @@ class XMLStackParserTests: XCTestCase {
             shouldProcessNamespaces: false
         ))
     }
+    
+    func testNestedMembers_removeWhitespaceElements() throws {
+        let parser = XMLStackParser(trimValueWhitespaces: false, removeWhitespaceElements: true)
+        let xmlData =
+            """
+            <SomeType>
+                <nestedStringList>
+                    <member>
+                        <member>foo</member>
+                        <member>bar</member>
+                    </member>
+                    <member>
+                        <member>baz</member>
+                        <member>qux</member>
+                    </member>
+                </nestedStringList>
+            </SomeType>
+            """.data(using: .utf8)!
+        let root = try! parser.parse(with: xmlData, errorContextLength: 0, shouldProcessNamespaces: false)
```

Comment:
No need to force try here, the test will rethrow the error, and `try!` is causing linter errors.

```suggestion
        let root = try parser.parse(with: xmlData, errorContextLength: 0, shouldProcessNamespaces: false)
```